iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/newtype.inl

```diff
@@ -107,6 +107,9 @@ inline NewType<T, Policies...>& NewType<T, Policies...>::operator=(T&& rhs) noex
     return *this;
 }
 
+/// @todo iox-#1554 Should operators + - * / ++ -- be added here with a static_assert for nicer error message (not done
```

Comment:
@elBoberido What's your take? It's not strictly needed, but if we decide to go this way I would do it for `Sortable`, too.

Reply:
@mossmaurice I'm not sure what you mean?

Reply:
```
static_assert(algorithm::doesContainType<newtype::Arithmetic<T>, Policies<T>...>(),
                  "This type is not {multipliable,dividable,addable,substractable}, please add the newtype::Arithmetic policy.");
```

could be added for a nicer error message. Same for the other traits.

Reply:
I'm still slow on the uptake :sweat_smile: 
Guess I have to look closer at the code

Reply:
@elBoberido That can be added later. I removed the todo.